regression-tests/pure2-return-tuple-no-identifier.cpp2

```diff
@@ -0,0 +1,3 @@
+d: () -> (int, int) = (1,2);
```

Comment:
This is intended to emit an error, and test filenames that should produce errors should have names ending with `-error.cpp2`.